collective/portlet/calendar/calendar.py

```diff
@@ -234,21 +273,25 @@ def getEventsForCalendar(self):
 
         if not self.options:
             # Folder, or site root.
-            # CalendarTool behavior is kept, but portlet options can change search results
             self.options['path'] = navigation_root_path
             if self.data.kw:
                 self.options['Subject'] = self.data.kw
-            if self.data.review_state:
-                self.options['review_state'] = list(self.data.review_state)
-        elif self.options and not self.options.get('review_state'):
-            # if using a Topic, we need to override the calendar default behaviour with review state
-            self.options['review_state'] = list(self.calendar.getCalendarStates())
+            self.options['review_state'] = list(self.data.review_state) \
```

Comment:
This list cast is not necessary since we untuple options in the end of this statement